coral/README.md

```diff
@@ -109,6 +109,13 @@ Coral uses the component library of Aiven's Aquarium design system:
 
 As a rule, please don't use css classes from the design system. All styles should be created by using the existing components and their properties.
 
+### Custom styles
+When we need custom styles - which should not be the case very often - we use [css modules](https://github.com/css-modules/css-modules). This enables us to add scoped css rules. Class names should be written in camelCases. 
```

Comment:
Nitpickiest nitpick: Typo? "camelCases" -> "camelCase"

Reply:
very much a typo! :D going to update that 🏃‍♀️